e2-studio-star-rx72n-firmware/tests/mocks/mock_adc_hal.c

```
@@ -296,14 +292,15 @@ rx_err_t adc_read_voltage_mv(adc_unit_t       unit,
   }
 
   /* Read raw ADC value */
+  uint16_t raw_value;
```

Clang (14.0.6)

[warning] variable 'raw_value' is not initialized

(cppcoreguidelines-init-variables)